sw/device/lib/crypto/include/datatypes.h

```diff
@@ -27,15 +27,18 @@ extern "C" {
  */
 typedef enum crypto_status {
   // Status is OK; no errors.
-  kCryptoStatusOK = 0xc6c5,
+  kCryptoStatusOK = 0xcb6,
```

Comment:
nit: is it possible to maintain OK as the generic ok number? `0x739`

Reply:
We can, although I'm worried it might not have as high a Hamming distance from the other status codes -- the hardened status generator doesn't have an option for one value being fixed afaik.

Reply:
I thought about this some more and modified the "OK" code to be `0x739`, then added unit tests to ensure the Hamming distance is sufficient.